code/pizzabot.js

```diff
@@ -1,7 +1,140 @@
+//Variables defining data
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
+
+const vegPic = document.getElementById("vegPizza");
+const pepPic = document.getElementById("pepPizza");
+const hawPic = document.getElementById("hawPizza");
+
+//--------------------------------------------------------------------------//
+
+//Sentances showing in html via the getElementById()
+// Greeting to the user from the pizzabot
+document.getElementById("greeting").innerHTML = `&#129302 Hi! Welcome to Claire & Claudia's pizza ordering service. I'm pizza bot and I'm here to help you with your order.`;
+
+//Showing the menu
+document.getElementById("menu").innerHTML = `On the menu we have: <li>${vegetarian}: ${pizzaPrice}kr</li> <li>${hawaiian}: ${pizzaPrice}kr</li> <li>${pepperoni}: ${pizzaPrice}kr</li>`;
+
+//Message to the user asking them to choose a pizza from the menu
+document.getElementById("choosePizza").innerHTML = "&#129302 Ok great. Let's get started with your order! Please write the name of the pizza you want to order:";
+
+//--------------------------------------------------------------------------//
+
+//Function compares if the parameter chosenPizza is true or false. Will return true if user inputs correct pizza name, otherwise will show message which is called further down in enteredPizzaValue
+const validateOrderName = (chosenPizza) => {
+    if (chosenPizza.toLowerCase() === vegetarian.toLowerCase() || chosenPizza.toLowerCase() === hawaiian.toLowerCase() || chosenPizza.toLowerCase() === pepperoni.toLowerCase()){
+        return true;
+    }
+    else { 
+        return false;
+    }
+};
```

Comment:
There are a number of ways to do this: 
```javascript
const validateOrderName = (chosenPizza) => {
    // To prevent doing `toLowerCase` three separate times
    const lowerCaseChoice = chosenPizza.toLowerCase();
  
    // Calculate the `boolean` value
    const isValidChoice = lowerCaseChoice === vegetarian.toLowerCase() || 
              lowerCaseChoice  === hawaiian.toLowerCase() || 
              lowerCaseChoice  === pepperoni.toLowerCase();
    return isValidChoice;
};
```
```Javascript
const validateOrderName = (chosenPizza) => { 
    // Single line return example showing that you can return the boolean directly
    return chosenPizza.toLowerCase() === vegetarian.toLowerCase() || 
               chosenPizza.toLowerCase() === hawaiian.toLowerCase() || 
               chosenPizza.toLowerCase() === pepperoni.toLowerCase();

};
```

Reply:
There are so many ways to write a simple function, but consider these alternatives and formulate an idea of what looks best, what's easiest to understand, and how to think about these types of functions in the future 👍